src/Common/tests/System/PlatformDetection.cs

```diff
@@ -43,6 +43,33 @@ public static partial class PlatformDetection
 
         public static int WindowsVersion => GetWindowsVersion();
 
+        public static bool IsAtLeast462()
+        {
+            if (!IsFullFramework)
+            {
+                return true;
```

Comment:
Sorry misread your statement. It will return true if !PlatformDetection.IsFullFramework, updating my PR then.